sky/cloud_stores.py

```diff
@@ -219,8 +220,76 @@ def get_storage_from_path(url: str) -> CloudStorage:
     return _REGISTRY[result.scheme]
 
 
+class IBMCosCloudStorage(CloudStorage):
+    """IBM Cloud Storage."""
+    # install rclone if package isn't already installed
+    # pylint: disable=line-too-long
```

Comment:
Can we remove this and shorten the long scripts? I believe all the scripts below can be shortened without breaking anything or conveying misinformation.

Reply:
Regarding the bug, fixed it by adding a call to `store_rclone_config` (which already creates .config/rclone/rclone.conf if it doesn't exist) to install rclone if not installed.